src/main/java/com/github/davidmoten/aws/maven/CloudFormationDeployer.java

```diff
@@ -210,15 +227,14 @@ private Status waitForCompletion(String stackName, int intervalMs, Log log) {
         String stackStatus = "Unknown";
         String stackReason = "";
 
-        log.info("waiting for action on  " + stackName);
+        log.info("waiting for action on " + stackName);
         long t = System.currentTimeMillis();
 
         while (true) {
             List<Stack> stacks;
             try {
                 stacks = cloudFormationClient.describeStacks(describeRequest).getStacks();
             } catch (AmazonCloudFormationException e) {
-                log.warn(e.getMessage());
```

Comment:
Why have you removed the warning?

Reply:
When stack is destroyed, in my tests it always throws exception here saying that stack is not found - being a bit misleading because this is exactly what was meant to happen. Perhaps it could be changed to `info`?

Reply:
info sounds fine if you could make that change

Reply:
Fixed.